pcsx2/gui/FrameForGS.cpp

```diff
@@ -591,11 +593,11 @@ void GSFrame::OnUpdateTitle( wxTimerEvent& evt )
 
 	const u64& smode2 = *(u64*)PS2GS_BASE(GS_SMODE2);
 
-	SetTitle( pxsFmt( L"%s | %ls (%ls) | Limiter: %ls | fps: %6.02f%ls | State %d",
+	SetTitle( pxsFmt( L"%s | %ls (%ls) | Limiter: %ls | Speed: %3d%% (%6.02f) %ls | State %d",
```

Comment:
Please remove the space after the `)`. `cpuUsage.c_str()` which follows it already starts with a space.